### ISSUE TITLE
ChromeOS Support

### ISSUE THREAD

**Issue by mikob** [go-vgo/robotgo#508]
- Robotgo version (or commit ref): v1.0.0-beta5.2
- Go version: 1.18.3
- Gcc version:
- Operating system and bit: ChromeOS 102.0.5005.75 (Official Build) (64-bit)
- Resolution: 2560x1280
- Can you reproduce the bug at [Examples](https://github.com/go-vgo/robotgo/blob/master/examples/main.go):
  - [ ] Yes (provide example code)
  - [ ] No
  - [X] Not relevant
- Provide example code:

```Go
package main

import (
	"github.com/go-vgo/robotgo"
)

func main() {
     robotgo.KeyTap("p", "ctrl");
}

```
- Log gist:

## Description
Tried using linux in ChromeOS (Crostini) and it does not work for simulating keyboard presses. Wonder if this is possible?

Error:
`load_input_helper [1883]: XkbGetKeyboard failed to locate a valid keyboard!`

...

**Comment by vcaesar** [comment]
Are you install linux X11 library?

**Comment by mikob** [comment]
No, didn't install any. I was under the impression that Crostini makes X11 possible. What X11 libraries are needed. Linux-enabled Chromebooks have apt-get.

**Comment by vcaesar** [comment]
https://github.com/go-vgo/robotgo#requirements.
